order/fragment.go

```diff
@@ -245,6 +245,11 @@ func (fragment *EncryptedFragment) Decrypt(privKey *rsa.PrivateKey) (Fragment, e
 	return decryptedFragment, nil
 }
 
+// IsNil checks if an EncryptedFragment is null.
+func (fragment *EncryptedFragment) IsNil() bool {
+	return fragment == nil || fragment.ID == (FragmentID{}) || len(fragment.ID) != 32 || len(fragment.OrderID) != 32
```

Comment:
len(fragment.ID) != 32 and  len(fragment.OrderID) != 32 will always be true as they are [32]byte